transports/janus_websockets.c

```diff
@@ -819,12 +818,17 @@ gboolean janus_websockets_is_admin_api_enabled(void) {
 int janus_websockets_send_message(janus_transport_session *transport, void *request_id, gboolean admin, json_t *message) {
 	if(message == NULL)
 		return -1;
-	if(transport == NULL || transport->transport_p == NULL || g_atomic_int_get(&transport->destroyed)) {
-		g_free(message);
+	if(transport == NULL || g_atomic_int_get(&transport->destroyed)) {
+		json_decref(message);
```

Comment:
Good catch here, very likely a leftover... it's in master as well, I'll fix that.